rpcs3/rpcs3qt/emu_settings.cpp

```diff
@@ -1235,9 +1235,8 @@ QString emu_settings::GetLocalizedSetting(const QString& original, emu_settings_
 		case CELL_SYSUTIL_LANG_ENGLISH_GB: return tr("English (UK)", "System Language");
 		case CELL_SYSUTIL_LANG_PORTUGUESE_BR: return tr("Portuguese (Brazil)", "System Language");
 		case CELL_SYSUTIL_LANG_TURKISH: return tr("Turkish", "System Language");
-		default:
-			break;
```

Comment:
removing this makes no sense

Reply:
It does nothing, at best it silences warning of missing enum values in the switch which you wouldn't want anyway.